doc/SpectatorView.Setup.md

```diff
@@ -43,7 +43,7 @@ To use the Spectator View codebase, its suggested to clone and reference the Mix
 
 ## Spatial Alignment Strategy Dependencies
 
-Spectator View requires multiple devices understanding a shared application origin in the physical world. In order to establish this shared application origin, you will need to choose and use a spatial alignment strategy. Different dependencies are needed for different strategies. For more information on spatial alignment strategies, see [here](../src/SpectatorView.Unity/Assets/SpatialAlignment/README.md).
+Spectator View requires multiple devices understanding a shared application origin in the physical world. In order to establish this shared application origin, you will need to choose and use a [spatial alignment strategy](../src/SpectatorView.Unity/Assets/SpatialAlignment/README.md). **The suggested spatial alignment strategy for Spectator View is Azure Spatial Anchors.** You will only need to go through one setup process below for the spatial alignment strategy you choose.

```

Comment:
>Spectator View is Azure Spatial Anchors [](start = 333, length = 39)

Consider adding a link directly to the ASA section of the SpatialAlignment/README.md that has stuff about Azure Spatial Anchors:

i.e.

```
[Azure Spatial Anchors](../src/SpectatorView.Unity/Assets/SpatialAlignment/README.md#azure-spatial-anchors-localization]
```

This is to reduce any room for doubt about if the person is reading the right section. Especially because it's further down, and isn't a top level section.

Reply:
these headers haven't appeared to work with docfx generation. I'll test out hyphens here today.

---
In reply to: [352932743](https://github.com/microsoft/MixedReality-SpectatorView/pull/290#discussion_r352932743) [](ancestors = 352932743)